mods/play_scenario/scenarios/gnaeus_pompeius_magnus.js

```diff
@@ -60,6 +59,7 @@
       },
       inheritance: 0,
       flagPlayScenarioModIsPompey: true,
+      flagFather:true,
```

Comment:
Make it `flagPlayScenarioModPompeyIsFather`

Reply:
done